### DISCUSSION TITLE
I've got some questions about the new Multiple GitHub accounts

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#143390] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I've just upgraded Visual Studio Code to version 1.95.1. I see that GitHub Copilot, which I have for my GitHub Pro license, now has the ability to have multiple GitHub accounts logged into it. I think this is a potentially excellent feature, but I can also see where this could cause problems. For example, I pay for my own GitHub Copilot, but my employer doesn't pay for any GitHub Copilot. (I've no idea if they ever will.) And I am 100% certain that they would be **EXTREMELY** upset if my personal GitHub account in VS Code "captures" anything from my work GitHub account. So, how aware is one version of GitHub account of other GitHub accounts logged into the same instance of Visual Studio Code?

**Comment by Akash1134** [comment, marked as answer]
Hi @rfalanga ,

Copilot is designed to respect the context of each account, meaning that interactions from your personal account shouldn't influence or "capture" any information from your work account. When using Copilot with multiple accounts in Visual Studio Code:

- Each GitHub account's data is isolated and private.
- Settings and usage data for Copilot will be scoped to the logged-in account. You need to ensure you switch accounts appropriately for different projects to avoid cross-account data usage.

For more details, please checkout the [GitHub Copilot Trust Center](https://copilot.github.trust.page/) 

Thanks!

**Reply by rfalanga** [reply]
Great advise, @Akash1134!

**Comment by AuroraEchos** [comment]
Hello! Regarding your concerns about using GitHub Copilot in VS Code, I can provide the following information:

First and foremost, GitHub Copilot is designed to respect users' privacy and data security. When you log into multiple GitHub accounts in VS Code, the data for each account is usually isolated. This means that the activities and data of one account are usually not shared or confused with another account.

However, to ensure that your work and personal data remain separate, it is recommended that you take some precautions when using GitHub Copilot:

1. ** Use separate VS Code instances ** : If you need to work on both work and personal projects, consider logging into your work and personal accounts in separate VS Code instances. This ensures data isolation and prevents potential cross-contamination.

2. ** Check account Settings regularly ** : Make sure your GitHub account Settings are correctly configured to protect your privacy and data. Check which apps and services have access to your account data and update your privacy Settings regularly.

**Reply by rfalanga** [reply]
Also, great advise @AuroraEchos!

**Comment by rfalanga** [comment]
I really appreciate what the two of you have said here. I wish I could mark both answers as the correct answer, but I can't. I'll mark the first answer as the answer, but really both are the answer.

**Comment by rfalanga** [comment]
I really appreciatre what you've said here
